exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,12 @@
+
+//Using the variables provided in the exercise calculate the percentage of mentors and students in the group
+
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var total =  numberOfStudents + numberOfMentors;
+var percentageOfStudents = (Math.round(numberOfStudents/total * 100));
```

Comment:
The outer `()` aren't needed.
It's best to leave spaces between mathematica ops.
```
Math.round(numberOfStudents / total * 100)
```